15-draft.json

```diff
@@ -18,6 +18,15 @@
       "description": "The name of your space",
       "type": "string"
     },
+    "type": {
```

Comment:
The key "type" is not ideal, because it's a reserved keyword in a lot of programming languages. What about "kind"?

Reply:
Resolving this, since we decided to remove the type anyways.